src/Tools/AI Test Toolkit/src/TestSuite/AITALTestSuiteMgt.Codeunit.al

```diff
@@ -232,17 +232,57 @@ codeunit 149037 "AIT AL Test Suite Mgt"
     /// <param name="DatasetFileName">The file name of the dataset file which will be used in the description of the dataset.</param>
     /// <param name="DatasetInStream">The InStream of the dataset file.</param>
     procedure ImportTestInputs(DatasetFileName: Text; var DatasetInStream: InStream)
+    var
+        CallerModuleInfo: ModuleInfo;
+    begin
+        NavApp.GetCallerModuleInfo(CallerModuleInfo);
+        ImportTestInputs(DatasetFileName, DatasetInStream, CallerModuleInfo, 0, DatasetFileName);
+    end;
+
+    /// <summary>
+    /// Import the Test Input Dataset from an InStream of a dataset in a supported format with a specific language.
+    /// Overwrite the dataset if the dataset with same filename is already imported by the same app
+    /// Error if the dataset with the same filename is created by a different app
+    /// </summary>
+    /// <param name="DatasetFileName">The file name of the dataset file which will be used in the description of the dataset.</param>
+    /// <param name="DatasetInStream">The InStream of the dataset file.</param>
+    /// <param name="LanguageID">The language ID to use for the dataset import.</param>
+    procedure ImportTestInputs(DatasetFileName: Text; var DatasetInStream: InStream; LanguageID: Integer)
+    var
+        CallerModuleInfo: ModuleInfo;
+    begin
+        NavApp.GetCallerModuleInfo(CallerModuleInfo);
+        ImportTestInputs(DatasetFileName, DatasetInStream, CallerModuleInfo, LanguageID, DatasetFileName);
+    end;
+
+
+    /// <summary>
+    /// Import the Test Input Dataset from an InStream of a dataset in a supported format with a specific language.
+    /// Overwrite the dataset if the dataset with same filename is already imported by the same app
+    /// Error if the dataset with the same filename is created by a different app
+    /// </summary>
+    /// <param name="DatasetFileName">The file name of the dataset file which will be used in the description of the dataset.</param>
+    /// <param name="DatasetInStream">The InStream of the dataset file.</param>
+    /// <param name="LanguageID">The language ID to use for the dataset import.</param>
+    /// <param name="Name">The name to use for the dataset import.</param>
+    procedure ImportTestInputs(DatasetFileName: Text; var DatasetInStream: InStream; LanguageID: Integer; Name: Text)
```

Comment:
For now it is OK, however we may need to use the builder or factory patterns. We should not be adding additional overloads.